.github/next_release.md

```diff
@@ -1,23 +1,36 @@
 We are pleased to announce the release 3.9.0 of the GUDHI library.
 
-As a major new feature, the GUDHI library now offers ...
+As a major new feature, the GUDHI library now offers a much faster implementation for the Cubical 2d case, fast cofaces access and stable simplex handles for the `Simplex_tree`, and a pip package for Python 3.12.
 
 We are now using GitHub to develop the GUDHI library, do not hesitate to [fork the GUDHI project on GitHub](https://github.com/GUDHI/gudhi-devel). From a user point of view, we recommend to download GUDHI user version (gudhi.3.X.X.tar.gz).
 
 Below is a list of changes made since GUDHI 3.8.0:
 
-- [Module](link)
-     - **...**
+- [Simplex_tree options](https://gudhi.inria.fr/doc/latest/struct_simplex_tree_options.html)
+     - A new option `link_nodes_by_label` to speed up cofaces and stars access, when set to true.
+     - A new option `stable_simplex_handles` to keep Simplex handles valid even after insertions or removals, when set to true.
 
-- [Module](link)
-     - **...**
+- [Simplex_tree](https://gudhi.inria.fr/doc/latest/group__simplex__tree.html)
+     - `Simplex_tree_options_full_featured` has been renamed `Simplex_tree_options_default` and `Simplex_tree_options_python`.
+     These are respectively the options used by all the utilities and by the python interface of the `SimplexTree` (as before this version).
+     - From GUDHI 3.9.0, `Simplex_tree_options_full_featured` now activates `link_nodes_by_label` and `stable_simplex_handles`.
```

Comment:
> The default parameter is now called `Simplex_tree_options_default` (same behavior as before), while `Simplex_tree_options_full_featured` now enables both `link_nodes_by_label` and `stable_simplex_handles`, making it slower. It remains recommended to use a custom option class that defines exactly the options you need (and derives from `Simplex_tree_options_minimal` for compatibility with future releases).

? (it isn't true that all utilities use "default", some use "fast")

Reply:
I precised when `Simplex_tree_options_default` was used by the `Simplex_tree` on [7b0f255](https://github.com/GUDHI/gudhi-devel/pull/1005/commits/7b0f255fe4f3ddf0aa04a7b18b873d73ac4ebf18)

Reply:
I wonder if we shouldn't include something like the speed warning of my paragraph above :thinking:

Reply:
@mglisse I added a speed warning on [d872578](https://github.com/GUDHI/gudhi-devel/pull/1005/commits/d872578d1768ac8d0257787ffe96e5636ed50a17)